### ISSUE TITLE
'outdated' - bullseye-slim - PHP 7.4

### ISSUE THREAD

**Issue by sincze** [domoticz/domoticz-docker#23]
Seems bullseye-slim comes by default with PHP 7.4.

I would like to use PHP 8.2 with Domoticz as such I presume default docker needs to be build against Bookworm will this be available soon?

**Comment by gizmocuz** [comment]
We are not able to do so because of the libssl version at the moment.
PHP is also not default installed in the docker
Are you having any script issues using PHP 7.4 or is it just because there is a new version you want to use this version?

**Comment by sincze** [comment]
I noticed ;-) , as I tried to create a PR for the Dockerbuild and ran into the libssl issue you mentioned indeed.  
I wanted to migrate my Domotica stack fully into docker.  
The PHP code currently is optimized for PHP 8.2 with deprecated functions of 7.4 removed.

So, currently playing around with the "customstart.sh" (great invention by the way)  by installing latest version in the container that way. Or maybe keep the NGINX and PHP in a separate container, also possible.
